site/guide/set-up-model-workflows.qmd

```diff
@@ -0,0 +1,213 @@
+
+## Configure workflow steps
+
+To add or edit workflow steps:
+
+1. [Log in to the {{< var vm_platform >}}]({{< var vm_url >}}).
+
+2. In the left sidebar, click [**{{< fa gear >}} Settings**]({{< var vm_url >}}/settings).
+
+3. Under Workplace Settings, select **{{< fa arrow-right-arrow-left >}} Workflows**.
+
+4. Under available workflows, select the process you'd like to add or edit the steps for:
+ 
+    - To add a step, drag and drop a new step onto the canvas. Hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears, then click on it and select **{{< fa gear >}} Configure**.
+    - To edit a step, hover over the step until the **{{< fa ellipsis-vertical >}}** menu appears. Then click on it and select **{{< fa gear >}} Configure**.
+
+5. After you're finished with step configuration, click **Save** to apply your changes before closing the configuration panel with **{{< fa x >}}**. 
+
+    ![](configure-steps.gif){fig-alt="A gif workflow step configuration."}
+
+    After you've configured a step, you can then [link your workflow together](#link-workflow-together).
+
+::: {.callout-tip appearance="minimal"}
+Refer to **[Available step types](#available-step-types)** for information on available step types and details on how to set up conditional requirements.
+:::
+
+### Available step types
+
+#### Status Change
+
+- Transitions a status. 
+- Requires [available resource statuses](customize-resource-statuses.qmd) associated with the workflow.
+
+| Field | Description |
+|---:|---|
+| When these conditions are met (optional) | [Add conditional requirements](#add-conditional-requirements) to qualify for this step.  |
+| Set status to (required) | Select the status to transition to. |
+: Status Change step configuration {.hover tbl-colwidths="[40,60]"}
+
+#### User Action
+- Creates a button that performs an action on behalf of the user. 
+- Requires [available resource statuses](customize-resource-statuses.qmd) associated with the workflow.
+
+| Field | Description |
+|---:|---|
+| Display action button when (optional) | [Add conditional requirements](#add-conditional-requirements) for displaying this step.  |
+| Action button label (required) | Text that displays on your action button. |
+| Request fields (optional) | Include [custom fields](manage-inventory-custom-fields.qmd) alongside your action if desired. Select the field, then click **+ Add Field** to insert.  |
+| To set status (required) | Select the [status](customize-resource-statuses.qmd) to transition to. |
+: User Action step configuration {.hover tbl-colwidths="[40,60]"}
+
+::: {.grid}
+
+::: {.g-col-7}
+![](configure-user-action.png){fig-alt="A screenshot showing a User Action step configuration with a request field of Date On Hold."}
+:::
+
+::: {.g-col-5}
+![](user-action-custom-field.png){fig-alt="A screenshot showing the action transition panel with the Date On Hold custom field."}
+
+::: {.callout-tip title="Example" appearance="minimal"}
```

Comment:
FYI, Quarto also supports putting callouts into the right margin which might help clean up a visually very busy section in the docs:

::: {.column-margin}
EXISTING_CALLOUT
:::

Reply:
Interesting! I will try that out and see how it looks.

Reply:
That doesn't look as good for these sets of images, so I'm going with the old version.